taskcluster/ci/build/kind.yml

```diff
@@ -114,3 +114,7 @@ overriden-jobs:
     release-lib-fetch-httpurlconnection:
         treeherder:
             symbol: 'lib-fetch-httpurlconnect(BR)'
+
+    feature-webcompat-reporter:
+        treeherder:
+            symbol: 'feature-webcompat-report(BN)'
```

Comment:
You can use `feature-webcompat-re(BN)` which is exactly 25-char-long. Any longer name will fail.

Reply:
I just tried that, and it still throws the same error. The other symbols all have the same lengths, `lib-fetch-httpurlconnect(BR)` for example, so I assumed the char length is without the parenthesis. :)

Either way, even with `feature-webcompat-re(BN)`, it's still complaining that the original name was too long:

> RuntimeError: Treeherder group and symbol names must not be longer than 25 characters: feature-webcompat-reporter(BN)

So I wonder if this is the right place to make this change?

Reply:
Ah, I see what's happening! You need to override the nightly and the release flavors too. They get automatically generated by this piece of code https://github.com/mozilla-mobile/android-components/blob/1fc9f540d49f939e9f932baf21f27a4748beb9ae/taskcluster/ac_taskgraph/loader/build_config.py#L29

Reply:
Ah, now I understand what's going on there! I did that, and I did define the group name in `taskcluster/ci/config.yml`, and now the decision task was happy. Great.

Thanks for your help, and sorry for the force-push mail-spam... :)

Reply:
I'm glad you're now unblocked! I'm sorry the error message wasn't crystal clear. I'm not sure how we can modify the stack to avoid this confusion later 🤔

Reply:
Not sure if there is a lot you can do, I think that's just a matter of understanding how taskcluster works, and where things are configured, and why.

I was confused that the link to `task-treeherder-config.yml` in the error message 404s (for me?), but I'm not sure if just seeing another yml file would have connected the strings in my brain, to be honest. But that's a thing I'll not do again, because it actually makes sense once you know what's going on.